application/forms/PreferenceForm.php

```diff
@@ -283,6 +294,36 @@ public function createElements(array $formData)
             )
         );
 
+        if (isset($formData['auto_refresh']) && $formData['auto_refresh']) {
+            $speeds = [1 => $this->translate('Default')];
+
+            foreach ([2, 4, 8] as $speed) {
+                // Using Form#translatePlural() not for $speed==1 and $speed!=1,
+                // but for different $speed-dependent plural forms, e.g. in Russian
+                $speeds[$speed] = sprintf($this->translatePlural('%dx slower', '%dx slower', $speed), $speed);
+                $speeds[rtrim(sprintf('%F', 1.0 / $speed), '0')] = sprintf(
+                    $this->translatePlural('%dx faster', '%dx faster', $speed),
+                    $speed
+                );
+            }
+
+            krsort($speeds);
```

Comment:
Please use the options as discussed in the issue:

```suggestion
            $speeds = [
                0.5 => $this->translate('Fast', 'refresh_speed'),
                ''  => $this->translate('Default', 'refresh_speed'),
                2.0 => $this->translate('Moderate', 'refresh_speed'),
                4.0 => $this->translate('Slow', 'refresh_speed')
            ];
```